--- a/src/spicelib/devices/cnt/cntdefs.h
+++ b/src/spicelib/devices/cnt/cntdefs.h
@@ -22,7 +22,6 @@ typedef struct sCNTinstance {
     struct sCNTinstance *CNTnextInstance;  /* pointer to next instance of
                                               *current model*/
     IFuid CNTname; /* pointer to character string naming this instance */
-    int CNTowner;  /* number of owner process */
     int CNTstates;     /* index into state table for this device */

     int CNTdNode;  /* number of the gate node of the cntfet */
